Description: Fix setup.py for old versions of pip
 Older versions of pip such as the one shipped with jessie does not support python version
 quantifiers in extra_requires. This function emulates the behaviour via install_requires.
 This patch can be removed once Debian Jessie is no longer supported.
Author: Mathias Ertl <mati@jabber.at>
Last-Update: 2017-05-07
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/setup.py
+++ b/setup.py
@@ -9,6 +9,10 @@
 except ImportError:
     from distutils.core import setup
 
+PY2 = sys.version_info[0] == 2
+install_requires = ['pycares>=1.0.0']
+if  sys.version_info[0] == 2:
+    install_requires.append('trollius')
 
 def get_version():
     return re.search(r"""__version__\s+=\s+(?P<quote>['"])(?P<version>.+?)(?P=quote)""", open('aiodns/__init__.py').read()).group('version')
@@ -21,11 +25,7 @@
       url              = "http://github.com/saghul/aiodns",
       description      = "Simple DNS resolver for asyncio",
       long_description = codecs.open("README.rst", encoding="utf-8").read(),
-      install_requires = ['pycares>=1.0.0'],
-      extras_require   = {
-          ':python_version=="3.3"': ['asyncio'],
-          ':python_version<="3.2"': ['trollius'],
-      },
+      install_requires = install_requires,
       packages         = ['aiodns'],
       platforms        = ["POSIX", "Microsoft Windows"],
       classifiers      = [
